tasks/sufia-user.rake

```diff
@@ -0,0 +1,14 @@
+namespace :sufia do
+  namespace :user do
+
+    desc "list user's email"
+    task "list_emails", [:file_name] => :environment do |cmd, args|
+      file_name = args[:file_name]
```

Comment:
If you were to make the meat of this task be `UserEmaiReport.new(file_name).write` it would be easier to test. ;)